docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -44,13 +44,15 @@ Note how changing the value of the start date section will call `onChange` even
 
 ## Pickers lifecycle
 
-### When is `onClose` called?
+### `onClose`
+
+#### When is `onClose` called?
 
 :::info
 In all the below scenarios, the picker closes when `onClose` is called, except if you are controlling the `open` prop.
 :::
 
-#### When the last view is completed
+##### When the last view is completed
```

Comment:
How do you feel about this h5?
To me, the resulting structure doesn't look the best. 🙈 
Maybe that is the reason why we don't have h5 anywhere in the docs? 🤔

Reply:
How do you think we could avoid it?

- Replace the current h5 with bullet points?
- Remove the current h2 and move its information to the current h3? 
- Keep the current h4 as direct child of the current h3 (do not add the "When is XXX called?" level)?
- something else?

Reply:
I'm in favor of 2nd or 3rd option, but 2nd one doesn't feel right given the current page structure (fields lifecycle, pickers lifecycle).
This technically leaves us with 3rd option, I don't have any better suggestion without refactoring the page structure. 🤔

Reply:
@LukasTy I applied 2 :+1: